src/Donation/DonationRequest.php

```diff
@@ -10,11 +10,23 @@
 use Misd\PhoneNumberBundle\Validator\Constraints\PhoneNumber as AssertPhoneNumber;
 use Ramsey\Uuid\Uuid;
 use Ramsey\Uuid\UuidInterface;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
 use Symfony\Component\Validator\Constraints as Assert;
 
+/**
+ * TODO: add endSubscriptionAt in the list when end subscription will be merged.
```

Comment:
I think it'll be better to add `status` property and ckeck it value instead of check if `subscription_ended_at is not null`

Reply:
Good idea, but I didn't want to do a bigger PR and maybe have an other PR for this refactor ?

Reply:
It could be done in this card maybe ?
https://app.clubhouse.io/enmarche/story/3967/etq-enmarche-je-souhaite-avoir-l-historique-de-toutes-les-transactions-paybox

Reply:
yes it was a suggestion ;)